cmd/erasure-server-pool.go

```diff
@@ -211,6 +211,8 @@ func (z *erasureServerPools) GetDisksID(ids ...string) []StorageAPI {
 func (z *erasureServerPools) GetRawData(ctx context.Context, volume, file string, fn func(r io.Reader, host string, disk string, filename string, info StatInfo) error) error {
 	found := 0
 	for _, s := range z.serverPools {
+		s.erasureDisksMu.RLock()
```

Comment:
Can we use s.getDisks() instead ?

I saw long disk calls or frozen disk syscalls in XFS before. If that is the case, this mutex will never be release. I think it is better to play it safe and use s.getDisks() instead.